tests/unit/test_charm.py

```diff
@@ -12,13 +12,81 @@ def setUp(self) -> None:
         self.harness.begin()
         self.maxDiff = None
```

Comment:
Is `self.MaxDiff` needed here? The only time it seems used is when you are initializing its value as `None`. Could probably delete since it looks like a holdover from Jon's parca tests.